pkg/compute/init.go

```diff
@@ -134,22 +310,39 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 		return fmt.Errorf("error reading package manifest: %w", err)
 	}
 
-	if c.name != "" {
-		fmt.Fprintf(progress, "Setting package name in manifest to %q...\n", c.name)
-		m.Name = c.name
+	fmt.Fprintf(progress, "Setting package name in manifest to %q...\n", c.name)
+	m.Name = c.name
+
+	if c.description != "" {
+		fmt.Fprintf(progress, "Setting description in manifest to %s...\n", c.description)
+		m.Description = c.description
 	}
 
-	if c.serviceID != "" {
-		fmt.Fprintf(progress, "Setting service ID in manifest to %q...\n", c.serviceID)
-		m.ServiceID = c.serviceID
+	if c.author != "" {
+		fmt.Fprintf(progress, "Setting author in manifest to %s...\n", c.author)
+		m.Authors = []string{c.author}
 	}
 
+	fmt.Fprintf(progress, "Setting service ID in manifest to %q...\n", service.ID)
+	m.ServiceID = service.ID
+
+	fmt.Fprintf(progress, "Setting version in manifest to 1...\n")
+	m.Version = 1
+
 	if err := m.Write(filepath.Join(c.path, ManifestFilename)); err != nil {
 		return fmt.Errorf("error saving package manifest: %w", err)
 	}
 
 	progress.Done()
-	text.Success(out, "Initialized package %s to %s", m.Name, abspath)
+
+	text.Break(out)
+
+	fmt.Fprintf(out, "Initialized package %s to:\n\t%s\n\n", text.Bold(m.Name), text.Bold(abspath))
+	fmt.Fprintf(out, "Manage this service at:\n\t%s\n\n", text.Bold(fmt.Sprintf("%s%s", manageServiceBaseURL, service.ID)))
+	fmt.Fprintf(out, "To compile the package, run:\n\t%s\n\n", text.Bold("fastly compute build"))
+	fmt.Fprintf(out, "To deploy the package, run:\n\t%s\n\n", text.Bold("fastly compute deploy"))
```

Comment:
Is the idiom of emitting "output" from a larger process by doing one or more

> Prompt:
>&nbsp;&nbsp;&nbsp;&nbsp;**Output**

blocks at the conclusion of the process done elsewhere in the CLI? If so, is there an opportunity to make this style consistent?

Reply:
Added a `text.Description()` helper in https://github.com/fastly/cli/pull/5/commits/97bdd8200a757642fb81ed9ee0ca1a8ece6a0b2a